src/App/Resources/AppResources.resx

```diff
@@ -1541,6 +1541,12 @@
   <data name="ThemeDefault" xml:space="preserve">
     <value>Default (System)</value>
   </data>
+  <data name="AutoDarkTheme" xml:space="preserve">
+    <value>Default Dark Theme</value>
+  </data>
```

Comment:
IMO, resource strings keys should be the same (when not too long) as the value to be easier to read in the code without the need to go to the resx file to check what is the value. So here I'd use `DefaultDarkTheme` as the key.
Furthermore if for some reason in the future we have a value "Auto Dark Theme" we couldn't use the key as it'd be used here.